libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -72,6 +84,28 @@ export class SkyAgGridWrapperComponent implements AfterContentInit, OnDestroy {
     }
     this.agGrid.gridReady.pipe(takeUntil(this.#ngUnsubscribe)).subscribe(() => {
       this.#moveHorizontalScroll();
+      this.agGrid.columnApi.getAllDisplayedColumns().forEach((column) => {
+        this.#updateInlineHelp(column);
+      });
+
+      [
+        Events.EVENT_COLUMN_MOVED,
+        Events.EVENT_COLUMN_RESIZED,
+        Events.EVENT_COLUMN_VISIBLE,
+        Events.EVENT_DISPLAYED_COLUMNS_CHANGED,
+        Events.EVENT_GRID_COLUMNS_CHANGED,
+        Events.EVENT_NEW_COLUMNS_LOADED,
+        Events.EVENT_VIEWPORT_CHANGED,
+        Events.EVENT_VIRTUAL_COLUMNS_CHANGED,
+      ].forEach((event) => {
+        fromEvent(this.agGrid.api, event)
+          .pipe(takeUntil(this.#ngUnsubscribe), debounceTime(10))
+          .subscribe(() => {
+            this.agGrid.columnApi.getAllDisplayedColumns().forEach((column) => {
+              this.#updateInlineHelp(column);
+            });
+          });
+      });
```

Comment:
AG Grid uses "DOM virtualization" to only draw rows and columns that are visible in the viewport, and it agressively destroys elements that are no longer on screen. This set of events is used to watch for changes where an inline help header may have been added or re-added to the DOM, using `debounceTime(10)` to prevent race conditions.